src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -543,3 +543,47 @@ def test_graph_required_access_e2e(self):
         finally:
             if app_id:
                 self.cmd('ad app delete --id ' + app_id)
+
+
+class ApplicationOptionalClaimsScenarioTest(ScenarioTest):
+
+    def test_application_optional_claims(self):
```

Comment:
Maybe consider moving the test to `ApplicationSetScenarioTest.test_application_set_scenario`?

Reply:
moved.